### PR TITLE
Handle excessive positional arguments in the CLI

### PR DESCRIPTION
**Pull request by roman-khimov**
The problem originally found by @fabwa (with voting command), it's not critical, but still we better prevent user errors where we can.

### CODE REVIEW

File: cli/smartcontract/smart_contract.go [hunk 1]
```diff
@@ -136,9 +136,10 @@ func NewCommands() []cli.Command {
 		Usage: "compile - debug - deploy smart contracts",
 		Subcommands: []cli.Command{
 			{
-				Name:   "compile",
-				Usage:  "compile a smart contract to a .nef file",
-				Action: contractCompile,
+				Name:      "compile",
+				Usage:     "compile a smart contract to a .nef file",
+				UsageText: "neo-go contract compile -i path [-o nef] [-v] [-d] [-m manifest] [-c yaml] [--bindings file] [--no-standards] [--no-events] [--no-permissions]",
```
Comment (AnnaShaleva) [inline cli/smartcontract/smart_contract.go:141]: Add `--bindings` flag?

File: cli/smartcontract/smart_contract.go [hunk 2]
```diff
@@ -393,9 +397,10 @@ func NewCommands() []cli.Command {
 				Usage: "manifest-related commands",
 				Subcommands: []cli.Command{
 					{
-						Name:   "add-group",
-						Usage:  "adds group to the manifest",
-						Action: manifestAddGroup,
+						Name:      "add-group",
+						Usage:     "adds group to the manifest",
+						UsageText: "neo-go contract manifest add-group -n nef -m manifest -a address -s address",
```
Comment (AnnaShaleva) [inline cli/smartcontract/smart_contract.go:402]: Add `-w wallet [--wallet-config path]`?

File: cli/wallet/wallet.go [hunk 3]
```diff
@@ -154,28 +158,31 @@ func NewCommands() []cli.Command {
...
 					decryptFlag,
 				},
 			},
 			{
-				Name:   "dump-keys",
-				Usage:  "dump public keys for account",
-				Action: dumpKeys,
+				Name:      "dump-keys",
+				Usage:     "dump public keys for account",
+				UsageText: "neo-go wallet dump-keys -w wallet [--wallet-config path] [-a address]",
```
Comment (AnnaShaleva) [inline cli/wallet/wallet.go:184]: Replace an old UsageText with `-w wallet [--wallet-config path]` for cases down below? E.g. for ` "export --wallet <path> [--decrypt] [<address>]"`.

File: cli/smartcontract/smart_contract.go (hunk 1 above)
Comment (roman-khimov) [reply cli/smartcontract/smart_contract.go:141]: It's there!

Comment (AnnaShaleva) [reply cli/smartcontract/smart_contract.go:141]: Oh, sorry, I've missed it.

File: cli/wallet/wallet.go (hunk 3 above)
Comment (roman-khimov) [reply cli/wallet/wallet.go:184]: Done.

File: cli/smartcontract/smart_contract.go (hunk 2 above)
Comment (roman-khimov) [reply cli/smartcontract/smart_contract.go]: Fixed!

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**